pypeit/par/pypeitpar.py

```diff
@@ -2234,6 +2234,17 @@ def __init__(self, telgridfile=None, sn_clip=None, resln_guess=None, resln_frac_
                                'must be downloaded from the GoogleDrive and installed in your PypeIt installation via ' \
                                'the pypeit_install_telluric script. NOTE: This parameter no longer includes the full ' \
                                'pathname to the Telluric Grid file, but is just the filename of the grid itself.'
+        
+        defaults['tell_npca'] = 5
+        dtypes['tell_npca'] = int
+        descr['tell_npca'] = 'Number of telluric PCA components used. Can be set to any number from 1 to 10.'
+        
+        defaults['teltype'] = 'pca'
+        dtypes['teltype'] = str
+        descr['teltype'] = 'Method used to evaluate telluric models, either pca or grid. The grid option uses a ' \
```

Comment:
It might also be good to instead print the valid options from the `valid_teltype()` definition. Just in case someone later adds to this.

Reply:
Done.